OpenUpSA/wazimap-ng-ui
# Minimise mapchip on initial page load

## Description
* Add label to show applied filters
* Show description icon if there is description
* Change icons

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-385

## How is it tested automatically?

## How should a reviewer test it locally
* Go to datamapper
* select any subindicator
* Mapchip should be minimised
* Mapchip header should show `x of x filters applied` label
* Mapchip header should show info icon only if there is description present.

## Screenshots
<img width="787" alt="Screenshot 2022-06-03 at 3 27 18 PM" src="https://user-images.githubusercontent.com/6871866/171832901-9a19eef4-9111-43e4-884f-886d8414fa49.png">


## Changelog

### Added

### Updated

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
